x-pack/plugins/apm/server/lib/rum_client/get_page_view_trends.ts

```diff
@@ -68,17 +68,15 @@ export async function getPageViewTrends({
       x: xVal,
       y: bCount,
     };
-    if (breakdownItem) {
-      const categoryBuckets = (bucket.breakdown as any).buckets;
-      categoryBuckets.forEach(
-        ({ key, doc_count: docCount }: { key: string; doc_count: number }) => {
-          if (key === 'Other') {
-            res[key + `(${breakdownItem?.name})`] = docCount;
-          } else {
-            res[key] = docCount;
-          }
+    if ('breakdown' in bucket) {
+      const categoryBuckets = bucket.breakdown.buckets;
+      categoryBuckets.forEach(({ key, doc_count: docCount }) => {
+        if (key === 'Other') {
+          res[key + `(${breakdownItem?.name})`] = docCount;
+        } else {
+          res[key] = docCount;
```

Comment:
@elastic/uptime Is this change ok with you? I cleaned up the elasticsearch types a bit and it broke this. Personally I think it's better now because it doesn't rely on `any`.

Reply:
The question is whether `if (breakdownItem)` is the same as `if ('breakdown' in bucket)`. To me it looks like it should be.

Reply:
Yes looks fine.